src/App.jsx

```diff
@@ -1,13 +1,155 @@
-
 import './App.css'
+import React, {useState} from "react";
+import Button from "./components/Button/Button.jsx";
+
 
 function App() {
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    const [sum, setSum] = useState({
+        aardbei: 0,
+        banaan: 0,
+        appel: 0,
+        kiwi: 0
+    });
+
+    function addButton(fruit) {
+        setSum(function (prevSum) {
+            let newSum = {
+                aardbei: prevSum.aardbei,
+                banaan: prevSum.banaan,
+                appel: prevSum.appel,
+                kiwi: prevSum.kiwi
+            };
+            newSum[fruit] = prevSum[fruit] + 1;
+            return newSum;
+        });
+    }
+
+    function subtractButton(fruit) {
+        setSum(function (prevSum) {
+            let newSum = {
+                aardbei: prevSum.aardbei,
+                banaan: prevSum.banaan,
+                appel: prevSum.appel,
+                kiwi: prevSum.kiwi
+            };
+            newSum[fruit] = prevSum[fruit] > 0 ? prevSum[fruit] - 1 : 0;
+            return newSum;
+        });
+    }
+
+    function handleReset() {
+        setSum({
+            aardbei: 0,
+            banaan: 0,
+            appel: 0,
+            kiwi: 0
+        });
+    }
+
+
+    const [formState, setFormState] = useState({
+        firstName: "",
+        lastName: "",
+        age: "",
+        postalCode: "",
+        remark: "",
+        deliverFrequency: "iedere week",
+        timeSlot: "Overdag",
+        terms: false
+    })
+
+    function handleChange(event) {
+        const changedFieldName = event.target.name;
+        const newValue = event.target.type === "checkbox" ? event.target.checked : event.target.value;
+
+        setFormState({
+            ...formState,
+            [changedFieldName]: newValue,
+        });
+    }
+
+    function handleSubmit(e) {
+        e.preventDefault();
+        console.log(sum);
+        console.log(formState);
+    }
+
+    return (
+        <>
+            <main>
+                <Button fruitTitle="🍓Aardbei">
```

Comment:
Ik zou een Counter component maken waar je de basis in vast zet waar je dus alleen de naam, de value, en de onclick meegeeft.